extension/intents/forms/forms.toml

```diff
@@ -26,6 +26,15 @@ match = """
 [[forms.focusNext.example]]
 phrase = "Focus next field"
 
+[forms.formSubmit]
+description = "Submit the form field"
+match = """
+  focus submit (form |)
```

Comment:
The word "focus" shouldn't be here. `Submit (form |)` should be enough.

Probably the local tests are failing right now in this case too (unfortunately I haven't figured out how to run these particular tests in CircleCI). Specifically the example you give below won't be matched by this pattern. You can test this by running `npm run jest`

Reply:
So what shall I change the example to so that we don't get CircleCI errors?

Reply:
No... if you change the match pattern and remove "focus" it should work.